etc/config/execution.defaults.properties

```diff
@@ -14,3 +14,8 @@ firejail.profile.wine=etc/firejail/wine.profile
 cewrapper=
 cewrapper.config.sandbox=etc/cewrapper/user-execution.json
 cewrapper.config.execute=etc/cewrapper/compilers-and-tools.json
+
+# Prefix execution with `stdbuf -o0` to have stdout output also for crashing executions.
+# Discussed further in https://github.com/compiler-explorer/compiler-explorer/issues/7239
+unbufferStdout=stdbuf
```

Comment:
Just a nit but maybe we can use unbufferStdoutExe or something to indicate its an executable that's expected?

Reply:
You're right, done.